modal.js

```diff
@@ -161,11 +161,10 @@ function openPopup() {
                             <img src="./images/g.png" alt="">
                             </button>
                         </a>
```

Comment:
- [ ] it would be better to make the description text on the mobile version a bit bigger as the one in the Figma design in order to match the design 
- Yours 
<img width="352" alt="Screenshot 2022-11-21 at 15 20 49" src="https://user-images.githubusercontent.com/61790587/203079305-14748a8d-f6d3-4a4e-8138-ae349f40c8a2.png">
- Figma 
<img width="323" alt="Screenshot 2022-11-21 at 15 24 46" src="https://user-images.githubusercontent.com/61790587/203079329-04650c31-cd64-41ca-ac72-1dc386e66495.png">

Reply:
Issue resolved. Thank you for drawing my attention